src/ads1x15/ads1x15.hpp

```diff
@@ -419,6 +432,7 @@ namespace upm {
             void updateConfigRegister(uint16_t update, bool read = false);
             uint16_t swapWord(uint16_t value);
 
+            mraa::MraaIo* mraaIo = NULL;
```

Comment:
See if you can manage without MraaIo pointer type.

Reply:
Done